src/Operators/laplacian_operators.jl

```diff
@@ -35,9 +35,9 @@ Calculates the Laplacian of c via
 which will end up at the location `ccc`.
 """
 @inline function ∇²(i, j, k, grid, c)
-    return 1/Vᵃᵃᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_∂xᶠᵃᵃ, c) +
-                                    δyᵃᶜᵃ(i, j, k, grid, Ay_∂yᵃᶠᵃ, c) +
-                                    δzᵃᵃᶜ(i, j, k, grid, Az_∂zᵃᵃᶠ, c))
+    return 1/Vᵃᵃᶠ(i, j, k, grid) * (δxᶠᵃᵃ(i, j, k, grid, Ax_∂xᶜᵃᵃ, c) +
+                                    δyᵃᶠᵃ(i, j, k, grid, Ay_∂yᵃᶜᵃ, c) +
+                                    δzᵃᵃᶠ(i, j, k, grid, Az_∂zᵃᵃᶜ, c))
```

Comment:
This only works for `Field{Face, Face, Face}`, right?

Reply:
Or rather `Field{Center, Center, Center}` since it's used to compute ∇²ϕ.

I'm actually still a little confused which convention we're following. More comments and documentation is needed as you pointed out a long time ago: https://github.com/CliMA/Oceananigans.jl/issues/696

Reply:
I believe the 3-symbol code refers to the location of the _result_. So as written, this calculates the Laplacian of a function located at `Face, Face, Face` (which also ends up at the location `Face, Face, Face`).